app/Http/Controllers/Admin/MenuController.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Models\Menu;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Rule;
+use Illuminate\Support\Facades\Validator;
+
+
+
+class MenuController extends Controller
+{
+    public function index()
+    {
+        $menus = Menu::orderBy('order')->get();
+        return view('admin.menus.index', compact('menus'));
```

Comment:
viết giống vây


        $agents = $this->agentService->getListAgent(true, $request->all());
        return view("agent.index", [
            "route" => "agents",
            "action" => "agent",
            "menu" => "menu-open",
            "active" => "active",
            "agents" => $agents,
        ]);